refresh jobs for downstream exposures (#7005)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud-integrations/orchestrate-exposures.md

---

> title: "Orchestrate exposures"

Author: ```suggestion
title: "Orchestrate downstream exposures"
```

sidebar_label: "Orchestrate exposures"
description: "Use dbt to proactively refresh the underlying data sources (like Tableau extracts) during scheduled dbt jobs."
image: /img/docs/cloud-integrations/auto-exposures/explorer-lineage2.jpg
---

> # Orchestrate exposures <Lifecycle status="enterprise,beta" />

Author: ```suggestion
# Orchestrate downstream exposures <Lifecycle status="enterprise,beta" />
```

<IntroText>

Use dbt [Cloud job scheduler](/docs/deploy/job-scheduler) to proactively refresh downstream exposures and the underlying data sources (extracts) that power your Tableau Workbooks.

</IntroText>

:::tip Available in private beta
Orchestrating exposures is currently available in private beta to dbt Cloud Enterprise accounts. To join the beta, contact your account representative.
:::

Orchestrating exposures integrates with [downstream exposures](/docs/cloud-integrations/downstream-exposures-tableau) and uses your `dbt build` job to ensure that Tableau extracts are updated regularly.

Control the frequency of these refreshes by configuring environment variables in your dbt environment.

<Expandable alt_header="Differences between visualizing and orchestrating downstream exposures">

> | Feature | Visualize downstream exposures | Orchestrate downstream exposures <Lifecycle status="beta"/> |

Reviewer: Should this table be the snippet:
```
import ActiveAutoExposuresOverview from '/snippets/_active-auto-exposures-overview.md';

<ActiveAutoExposuresOverview />
```

| ---- | ---- | ---- |
| Purpose | Automatically brings downstream assets into your dbt lineage. | Proactively refreshes the underlying data sources during scheduled dbt jobs. |
| Benefits | Provides visibility into data flow and dependencies. | Ensures BI tools always have up-to-date data without manual intervention. |
| Location  | Exposed in [dbt Explorer](/docs/collaborate/explore-projects) | Exposed in [dbt Cloud scheduler](/docs/deploy/deployments) |
| Supported BI tool | Tableau | Tableau |
| Use case | Helps users understand how models are used and reduces incidents. | Optimizes timeliness and reduces costs by running models when needed. |
</Expandable>

## Prerequisites

To orchestrate downstream exposures, you should meet the following:

> - [Configured downstream exposures](/docs/cloud-integrations/downstream-exposures-tableau) and the desired exposures are included in your lineage.

Reviewer: ```suggestion
- [Configured downstream exposures](/docs/cloud-integrations/downstream-exposures-tableau) and ensured desired exposures are included in your lineage
```

> - Your environment and jobs are on a supported [release track](/docs/dbt-versions/cloud-release-tracks) dbt.

Reviewer: ```suggestion
- Verified your environment and jobs are on a supported dbt [release track](/docs/dbt-versions/cloud-release-tracks).
```

- Have a dbt Cloud account on the [Enterprise plan](https://www.getdbt.com/pricing/).

> - Have set up a [production](/docs/deploy/deploy-environments#set-as-production-environment) deployment environment for each project you want to explore, with at least one successful job run. 

Reviewer: ```suggestion
- Created a [production](/docs/deploy/deploy-environments#set-as-production-environment) deployment environment for each project you want to explore, with at least one successful job run.
```
